docs/src/test/java/io/improbable/snippet/SequenceExample.java

```diff
@@ -112,10 +114,26 @@ public Sequence buildSequenceTimeSeries() {
 
         Sequence sequence = new SequenceBuilder<Integer>()
             .withInitialState(dictionary)
+            .withIdentifyingNamespace("Keanu-Example")
             .count(5)
             .withFactory(factory)
             .build();
 
+        //We can now do a few things:
+        Vertex<?> seedVertex = sequence.getLastItem().get(x1Label);
```

Comment:
So I guess I've used it to mean "an arbitrary vertex in the graph, from which we can get the connected graph and construct the bayes net".
Agreed this may be confusing although I think `sequenceOutput` could also be confusing because it's actually not relevant that it's an output in this case.
Perhaps we should even add a `.getBayesNet` method to Sequence?? What do you think?

Reply:
Yes, I think `toBayesNet` would be a useful method.

We already have `model.to_bayes_net()` in python so this would tie in well with that.